src/main/resources/logback-spring.xml

```diff
@@ -4,9 +4,9 @@
 
 	<!-- Logger for sensitive data -->
 	<appender name="secureLog" class="ch.qos.logback.core.rolling.RollingFileAppender">
-		<file>/secure-logs/secure.log</file>
+		<file>secure-logs/secure.log</file>
```

Comment:
Tror vi trenger `/secure-logs/secure.log` her
https://doc.nais.io/observability/logs/#log-files

Reply:
jeg la den tilbake, og la til en egen logback for "local"